erc20-connector/contracts/AdminControlled.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.6.12;
+
+contract AdminControlled {
```

Comment:
Don't copy past this code and reuse the one from `rainbow-bridge`

Reply:
I'd like to but this part hasn't been merged yet.